tests/test_main.py

```diff
@@ -1,31 +1,67 @@
+from curses import panel
+from urllib import response
+
+from app.main import app, knowledge_panel
 from fastapi.testclient import TestClient
-import requests
-from app.main import app
 
 client = TestClient(app)
 
-def test_answer_questions_brand_president():
-    response = client.get("/brand/president")
+
+def test_hello():
+    response = client.get("/")
     assert response.status_code == 200
-    assert response.json() == {"knowledge_panels": [
-                {
-                    "hunger-game": {
-                        "type": "hunger-game",
-                        "level": "questions",
-                        "elements": [
-                            {
+    assert response.json() == {
+        "message": "Hello from facets-knowledge-panels! Tip: open /docs for documentation"
+    }
+
+
+def test_knowledge_panel():
+    response = client.get("/knowledge_panel")
+    assert response.status_code == 422
```

Comment:
Please also assert the content of response. (should be an empty list of knowledge panels)